tensorflow/python/ops/image_ops_impl.py

```diff
@@ -766,32 +766,32 @@ def _get_dim(tensor, idx):
 def pad_to_bounding_box(image, offset_height, offset_width, target_height,
                         target_width):
   """Pad `image` with zeros to the specified `height` and `width`.
-
   Adds `offset_height` rows of zeros on top, `offset_width` columns of
   zeros on the left, and then pads the image on the bottom and right
   with zeros until it has dimensions `target_height`, `target_width`.
-
   This op does nothing if `offset_*` is zero and the image already has size
   `target_height` by `target_width`.
-
+  Usage example:
+    
+    >>> x = tf.random.uniform((28, 28, 1))
+    >>> tf.image.pad_to_bounding_box(x, 28, 28, 100, 100).shape
+    TensorShape([100, 100, 1])
```

Comment:
I would rather have `x` be a small constant with some nice values and then pad to a slightly larger box (instead of going from 28 to 100) and displaying the values instead of just the shape

Reply:
you mean `tf.constant`?

Reply:
Yes

Reply:
Hi! I have no problem with the changing of the data, I just dont know how to display the values of a tensor. nvm I'll find out😊

Reply:
Just run the test on your local machine. https://www.tensorflow.org/community/contribute/docs_ref#test_on_your_local_machine

Reply:
This is still not done